build.gradle

```diff
@@ -75,6 +96,29 @@ jar {
     }
 }
 
+tasks.register("runServer", JavaExec) {
+    dependsOn "jar"
+
+    mainClass = "net.raphimc.viaproxy.ViaProxy"
+    classpath sourceSets.main.runtimeClasspath
+
+    workingDir = file("run")
+
+    doFirst {
+        def pluginsDir = new File(workingDir, "plugins")
+        if (!pluginsDir.exists())
+            pluginsDir.mkdirs()
+
+        // Copy JAR into plugins dir
+        new File(pluginsDir, "plugin.jar").bytes = tasks.jar.archiveFile.get().asFile.bytes
+    }
+
+    doLast {
+        // Delete JAR from plugins dir
+        new File(workingDir, "plugins/plugin.jar}").delete()
```

Comment:
Woops! I had that set to interpolate with `project.name`, then decided to hardcode it with *something*. I was one character short when replacing it seems. It's not it's first time, I had to fix the interpolation because I had an extra `}`.